src/Client/ClientBase.cpp

```diff
@@ -47,6 +47,10 @@
 #include <Parsers/ASTQueryWithOutput.h>
 #include <Parsers/ASTLiteral.h>
 #include <Parsers/ASTIdentifier.h>
+#include <Parsers/ASTColumnDeclaration.h>
+#include <Parsers/ASTIndexDeclaration.h>
+#include <Parsers/ASTConstraintDeclaration.h>
+#include <Parsers/ASTProjectionDeclaration.h>
```

Comment:
Why do we need anything except `ASTColumnDeclaration.h` here?

Reply:
Nice catch, that was leftovers from the first version.